test/linarith.lean

```diff
@@ -40,6 +40,14 @@ example (A B : Rat) (h : 0 < A * B) : 0 < 8*A*B := by
 example (A B : Rat) (h : 0 < A * B) : 0 < A*8*B := by
   linarith
 
+example [LinearOrderedCommRing α] (x : α) : 0 ≤ x := by
+  have h : 0 ≤ x := sorry
```

Comment:
```suggestion
example [LinearOrderedCommRing α] (x : α) (h : 0 ≤ x) : 0 ≤ x := by
```

Reply:
@gebner The bugfix tests the difference between these two things -- your version already passed.

Reply:
:bulb: Oh, they're testing the `withMainContext`!  That's cryptic..